js/articleView.js

```diff
@@ -61,6 +63,8 @@
         $('#title-filter').append(a.toHtml('#title-filter-template'));
       };
     });
+    // TODO I wanted to add this to the JSON page but couldn't comment. I want to change the information in my projects section. I want them to actually reflect the projects I have created so far in my coding. RIght now I have photos and some filler information but I would like it to actually reflect the projects I have worked on.
```

Comment:
As you've mentioned and know already, I think we are at the point in our portfolio where finding that relevant data, what articles you want to present, etc, will be important for the Portfolio to take shape now that we have a lot of the functionality. I have to go through this process still myself. I like the idea you mentioned previously about using the counter to count/show the skills used in each project, or time spent to finish the project.